boards/arty_e21/src/main.rs

```diff
@@ -44,6 +43,12 @@ static mut PROCESS_PRINTER: Option<&'static capsules_system::process_printer::Pr
 
 kernel::stack_size! {0x1000}
 
+struct ProcessManagementCapabilityObj {}
+unsafe impl capabilities::ProcessManagementCapability for ProcessManagementCapabilityObj {}
```

Comment:
It feels like we should have a macro for forging a capability type, without instantiating it, no?

This above is problematic, at least in the sense that you can now create an arbitrary number of capability objects safely in the board crate (which is not a deal breaker for this PR right now, but not great either). Capability structs should have a private field, and an unsafe constructor.

Reply:
I'm not sure I understand the problem here. Boards create capabilities and can create as many as they want. So what are you suggesting is the issue?

Reply:
This is a problem with the "no unsafe in components" idea, the type needs to be used in multiple places making using our macro not viable.